ide/coqide/coqOps.ml

```diff
@@ -402,7 +402,20 @@ object(self)
          compare and compensate for the shift. *)
       let offset, _line_shift = offset_compensation script#buffer s in
       let ss = find_all_tooltips s (iter#offset - offset) in
+      let ss = if ss = [] then Doc.get_errors document else ss in
       let msg = String.concat "\n" (CList.uniquize ss) in
+      let markup_escape msg =
+        let buf = Buffer.create (String.length msg) in
+        String.iter (fun c -> match c with
+          | '\'' -> Buffer.add_string buf "&#39;"
+          | '<' -> Buffer.add_string buf "&#60;"
+          | '>' -> Buffer.add_string buf "&#62;"
+          | '"' -> Buffer.add_string buf "&#34;"
+          | c -> Buffer.add_char buf c
+        ) msg;
```

Comment:
Is this general-purpose and worth to be added as a "util"?

Reply:
The usual escapes for HTML are `"&'<>`, I will update.  That makes it general purpose.  Shall I put it in Ideutils or Util?

Reply:
I would say in `clib/cString.ml`, or, if considered too specific by some of us, in `ideutils.ml`.

Reply:
Done.